model/task_queue_test.go

```diff
@@ -305,3 +305,27 @@ func TestTasQueueGenerationTimes(t *testing.T) {
 	assert.True(ok)
 	assert.Equal(now, genTime)
 }
+
+func TestClearTaskQueue(t *testing.T) {
+	assert := assert.New(t)
+	distro := "distro"
+	tasks := []TaskQueueItem{
+		{
+			Id: "task1",
+		},
+		{
+			Id: "task2",
+		},
+		{
+			Id: "task3",
+		},
+	}
```

Comment:
This tests that it does at least what we expect, but not that it does at most what we expect. It might be worth saving an additional distro, and make sure that that one is not deleted by the route.